mediacapture-streams/enumerating-local-media-devices/api-present.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+<head>
+<title>enumerateDevices: test that enumerateDevices is present</title>
+<link rel="author" title="Dr Alex Gouaillard" href="mailto:agouaillard@gmail.com"/>
+<link rel="help" href="https://w3c.github.io/mediacapture-main/#enumerating-devices">
+<meta name='assert' content='Check that the enumerateDevices() method is present.'/>
+<link rel='stylesheet' href='/resources/testharness.css' media='all'/>
+</head>
+<body>
+<h1 class="instructions">Description</h1>
+<p class="instructions">This test checks for the presence of the
+<code>navigator.mediaDevices.enumerateDevices()</code> method.</p>
+<div id='log'></div>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+"use strict";
+//NOTE ALEX: for completion, a test for ondevicechange event is missing.
+test(function () {
+  assert_true(undefined !== navigator.mediaDevices.enumerateDevices(), "navigator.mediaDevices.enumerateDevices() exists");
```

Comment:
I think @foolip mentioned it before, but it seems to me more logical to test 
```js
undefined !== navigator.mediaDevices.enumerateDevices
```
since we're checking for existence